launch/config.ts

```diff
@@ -180,7 +181,8 @@ const config = {
       image: 'parallelfinance/polkadot-collator:v0.9.12',
       chain: 'shell',
       ctokenId: 4000,
-      cap: '100000000000000000',
+      cap: '100000000000000',
```

Comment:
Why this cap is changed?

Reply:
@i5possible because we would like to make it easier for testing the edge case

Reply:
So we are not goint to deploy it to prod and just for testing usage?

Reply:
this script is only made for testing purpose because on prod we cannot control relaychain at all. It's up to parity